### PR TITLE
Fixed #24940 -- Made model managers hashable

### PR DESCRIPTION
**Pull request by MarkusH**
https://code.djangoproject.com/ticket/24940

### CODE REVIEW

File: docs/releases/1.8.3.txt [hunk 1]
```diff
@@ -74,3 +74,6 @@ Bugfixes
 
 * Fixed crash when uploading images with MIME types that Pillow doesn't detect,
   such as bitmap, in ``forms.ImageField`` (:ticket:`24948`).
+
+* Fixed a regression when deleting a model that has a ``GenericRelation`` and a
```
Comment (timgraham) [inline docs/releases/1.8.3.txt]: and a -> with a

Comment (timgraham) [inline docs/releases/1.8.3.txt]: deleting in a model through the admin

Comment (MarkusH) [inline docs/releases/1.8.3.txt]: "deleting in a model through the admin" doesn't make sense in my ears :stuck_out_tongue_winking_eye:

Comment (timgraham) [inline docs/releases/1.8.3.txt]: mine neither; chop "in".

**Comment by MarkusH** [comment]
Merged in d3d66d47222dd8765a20a15fdc754c0ed7635404

**Pull request closed without merging**